sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+This technical-elemental coupling uniquely makes visible, in a way that might not be possible without such an a video apparatus, the otherwise invisible mutability and bidirectional flux of time. Or perhaps this system actually brings into being--makes real--a new kind of time via its mediations and modulations?
+
+The effect can be disorienting. As a result of _Tofino_'s temporal phasing, cuts occur at different times in one layer than in the other, causing a new shot to be dizzyingly overlaid on top of the previous one for a period. Throughout, the camera is resting directly on a wooden ledge and is frequently buffeted the strong winds, causing it to shake and shift its position. At these moments, the layers are further deregistered, emphasizing the phasing and doubling effects and evoking a disorienting "double vision" effect that hints at the proprioceptive ambiguity of Ernie Gehr's Side/Walk/Shuttle discussed earlier.
+
+![A video still from Tofino, showing a composited view of Wikaninnish Beach.](../images/video-stills/tofino.png)
+_Figure x: A video still from_ Tofino.
+
+Jean Epstein, writing in 1946, imagined that the cinema's mechanical-perceptual apparatus, with its strange capacity to materialize time, represented a new form thinking: "This machine that stretches or condenses duration, demonstrating the variable nature of time, preaching the relativity of all measures... Without it, we would see nothing of what time might feel like materially... [it is] a form of thinking by the rules of analysis and synthesis that, without the cinematographic apparatus, humans would have been incapable of implementing" (18). Epstein's "intelligence of the machine," envisions cinema as a transformational technics that couples with human perception. As Bernard Stiegler later describes it, "cinema weaves itself into our time; it becomes the temporal fabric of... unconscious consciousness" (11). Here, however, the nature of the system and its slow and methodical unfolding suggest a model of cinematic time that is not just coextensive with our own time, but actively modulatory of it; the temporal fabric of a "conscious consciousness" of active and self-aware viewing. I will pick this thread up again in the next section when discussing _In Passing_.
```

Comment:
Comment from David: "This should be a footnote: 'I will pick this thread up again in the next section when discussing In Passing.'"